src/scss/_layout-card.scss

```diff
@@ -0,0 +1,35 @@
+.ucf-news.card-layout {
+
+  .ucf-news-thumbnail-image {
+    height: auto;
+    max-width: 100%;
+  }
+
+  @media (min-width: 576px) {
+
+    .card-deck {
```

Comment:
I'd lean toward making these card classes unique/not matching Athena's classes, just for the sake of not accidentally causing style conflicts with future Athena updates.